packages/design-tokens/sass/spacing.scss

```diff
@@ -1,5 +1,53 @@
 /** THIS IS AN AUTOGENERATED FILE **/
 
+$spacing-0: var(--spacing-0, 0);
+$spacing-1: var(--spacing-1, 0.0625rem);
+$spacing-2: var(--spacing-2, 0.125rem);
+$spacing-4: var(--spacing-4, 0.25rem);
+$spacing-6: var(--spacing-6, 0.375rem);
+$spacing-8: var(--spacing-8, 0.5rem);
+$spacing-12: var(--spacing-12, 0.75rem);
+$spacing-16: var(--spacing-16, 1rem);
+$spacing-24: var(--spacing-24, 1.5rem);
+$spacing-32: var(--spacing-32, 2rem);
+$spacing-40: var(--spacing-40, 2.5rem);
+$spacing-48: var(--spacing-48, 3rem);
+$spacing-56: var(--spacing-56, 3.5rem);
+$spacing-64: var(--spacing-64, 4rem);
+$spacing-72: var(--spacing-72, 4.5rem);
+$spacing-80: var(--spacing-80, 5rem);
+$spacing-96: var(--spacing-96, 6rem);
+$spacing-112: var(--spacing-112, 7rem);
+$spacing-128: var(--spacing-128, 8rem);
+$spacing-160: var(--spacing-160, 10rem);
+$spacing-200: var(--spacing-200, 12.5rem);
+$spacing-240: var(--spacing-240, 15rem);
+$spacing-280: var(--spacing-280, 17.5rem);
+$spacing-320: var(--spacing-320, 20rem);
+$spacing-0-id: --spacing-0;
```

Comment:
These look strange 🤔

Reply:
Yeah agreed. But this file is autogenerated and the previous shirt sizes get the `-id` treatment too for whatever reason.

Reply:
Ohhhh they do. I completely missed it since they were slotted in between. I wonder why these ones don't slot between each other like the others do 🤔

Reply:
hmmm maybe numbers have a strange sequential sorting logic? In any case, looks legit 🤷 RE: why they output `-id` I would assume it's some kind of convention around using global and locally scoped CSS variables, though that's a stab in the dark